tests/test_workspace.py

```diff
@@ -36,6 +36,15 @@ def workspace_factory(workspace_xml):
     return lambda: pyhf.Workspace(workspace_xml)
 
 
+@pytest.fixture
```

Comment:
```suggestion
@pytest.fixture(scope="module")
```

I _think_ this can safely be scoped to module.